examples/regen.rs

```diff
@@ -0,0 +1,20 @@
+//
+// A simple regeneration script for PB stubs and skeletons
+//
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
would it make more sense to put this into `build.rs` so it regenerates automatically on build?

Reply:
Not at this time. The issue is we want interop with otel collector and this tracks `main` it isn't pinned to a branch.
Currently - `tonic`, `prost` and the proto buffers are all moving targets. So I'd prefer to track it manually